src/main/java/mSearch/daten/ListeFilme.java

```diff
@@ -275,18 +269,18 @@ public synchronized void setMeta(ListeFilme listeFilme) {
         System.arraycopy(listeFilme.metaDaten, 0, metaDaten, 0, MAX_ELEM);
     }
 
-    public synchronized DatenFilm istInFilmListe(String sender, String thema, String titel) {
-        // prüfen ob es den Film schon gibt
-        // und sich evtl. nur die URL geändert hat
-        for (DatenFilm film : this) {
-            if (film.arr[DatenFilm.FILM_SENDER].equals(sender)
-                    && film.arr[DatenFilm.FILM_THEMA].equalsIgnoreCase(thema)
-                    && film.arr[DatenFilm.FILM_TITEL].equalsIgnoreCase(titel)) {
-                return film;
-            }
-        }
-        return null;
-    }
+//    public synchronized DatenFilm istInFilmListe(String sender, String thema, String titel) {
+//        // prüfen ob es den Film schon gibt
+//        // und sich evtl. nur die URL geändert hat
+//        for (DatenFilm film : this) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)